bolts/data/datasets/vision/celeba.py

```diff
@@ -149,11 +148,11 @@ def __init__(
         )
 
         x = np.array(attrs.index)
-        s = torch.as_tensor(attrs[subclass.name].to_numpy())
-        y = torch.as_tensor(attrs[superclass.name].to_numpy())
+        _s = torch.as_tensor(attrs[subclass.name].to_numpy())
```

Comment:
Why?

Reply:
I also don't like reassigning variables if it can be helped